From cd46d55e4a82c70beba5313d7de3d63c93623dc2 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Wed, 25 May 2016 14:41:10 +0300
Subject: [PATCH 27/46] ARM: dts: imx7d: cl-som-imx7: update defconfig

Update defconfig to enable WiLink8 and USB-to-serial support.
Enable:
        WL_TI
        WLCORE
        WLCORE_SDIO
As module:
        WL18XX
        USB_ACM
        USB_SERIAL
        USB_SERIAL_GENERIC
        USB_SERIAL_BELKIN
        USB_SERIAL_CP210X
        USB_SERIAL_CYPRESS_M8
        USB_SERIAL_FTDI_SIO
        USB_SERIAL_MCT_U232
        USB_SERIAL_MOS7720
        USB_SERIAL_MOS7840
        USB_SERIAL_PL2303
Disable:
        BRCMFMAC

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/configs/cl_som_imx7_defconfig | 16 +++++++++++++++-
 1 file changed, 15 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/cl_som_imx7_defconfig b/arch/arm/configs/cl_som_imx7_defconfig
index 6bbea9f..42d2c05 100644
--- a/arch/arm/configs/cl_som_imx7_defconfig
+++ b/arch/arm/configs/cl_som_imx7_defconfig
@@ -127,7 +127,10 @@ CONFIG_SMC911X=y
 CONFIG_SMSC911X=y
 # CONFIG_NET_VENDOR_STMICRO is not set
 CONFIG_AT803X_PHY=y
-CONFIG_BRCMFMAC=m
+CONFIG_WL_TI=y
+CONFIG_WL18XX=m
+CONFIG_WLCORE=y
+CONFIG_WLCORE_SDIO=y
 # CONFIG_INPUT_MOUSEDEV_PSAUX is not set
 CONFIG_INPUT_EVDEV=y
 CONFIG_INPUT_EVBUG=m
@@ -219,10 +222,21 @@ CONFIG_SND_SOC_IMX_MC13783=y
 CONFIG_USB=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_MXC=y
+CONFIG_USB_ACM=m
 CONFIG_USB_STORAGE=y
 CONFIG_USB_CHIPIDEA=y
 CONFIG_USB_CHIPIDEA_UDC=y
 CONFIG_USB_CHIPIDEA_HOST=y
+CONFIG_USB_SERIAL=m
+CONFIG_USB_SERIAL_GENERIC=y
+CONFIG_USB_SERIAL_BELKIN=m
+CONFIG_USB_SERIAL_CP210X=m
+CONFIG_USB_SERIAL_CYPRESS_M8=m
+CONFIG_USB_SERIAL_FTDI_SIO=m
+CONFIG_USB_SERIAL_MCT_U232=m
+CONFIG_USB_SERIAL_MOS7720=m
+CONFIG_USB_SERIAL_MOS7840=m
+CONFIG_USB_SERIAL_PL2303=m
 CONFIG_NOP_USB_XCEIV=y
 CONFIG_USB_MXS_PHY=y
 CONFIG_USB_GADGET=y
-- 
1.9.1

